app/views/layouts/_header.html.erb

```diff
@@ -175,38 +181,101 @@
   <%= render :partial => "layouts/alerts" %>
 </div>
 
-<!-- Login Modal -->
+<!-- Login/Signup Modal -->
 <div class="modal fade" id="loginModal" tabindex="-1" role="dialog" aria-hidden="true">
-  <div class="modal-dialog modal-dialog-centered" role="document">
+  <div class="modal-dialog" role="document">
     <div class="modal-content">
       <div class="modal-body">
         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
-        <%= render :partial => "user_sessions/new" %>
+        
+        <div class="clearfix">
+          <div id="signupContainer">
+            <%= render :partial => "users/form2" %>
+          </div>
+          <div id="loginContainer">
+            <%= render :partial => "user_sessions/new" %>
+          </div>
+        </div>
       </div>
     </div>
 
   </div>
 </div>
 
-<style>
-.modal {
-  text-align: center;
-  padding: 0!important;
-}
+<!-- Modal jquery -->
+<script>
+<<<<<<< HEAD
```

Comment:
Please solve this conflict and all other conflicts

Reply:
Sorry about that! I've fixed the merge conflicts

Reply:
@JonathanXu1 there are many other changes suggested to you in this pr. Also, I listed some suggestions. Please complete them and then ping me to review.

Reply:
Thanks for resolving conflicts.